src/node_worker.h

```diff
@@ -80,14 +80,13 @@ class Worker : public AsyncWrap {
 
   MultiIsolatePlatform* platform_;
   v8::Isolate* isolate_ = nullptr;
-  uv_thread_t tid_;
+  std::optional<uv_thread_t> tid_;  // Set while the thread is running
 
   std::unique_ptr<InspectorParentHandle> inspector_parent_handle_;
 
   // This mutex protects access to all variables listed below it.
   mutable Mutex mutex_;
 
-  bool thread_joined_ = true;
```

Comment:
The comment here implied that this was protected by the mutex, but a) that didn’t reflect the actual usage of this field and b) it’s only ever accessed from the parent thread anyway, so no locking is necessary here